### PR TITLE
Refs #33476 -- Refactored problematic code before reformatting by Black.

### PR DESCRIPTION
**Pull request by felixxm**
In these cases `Black` produces unexpected results, e.g.
```python
def make_random_password(
    self,
    length=10,
    allowed_chars='abcdefghjkmnpqrstuvwxyz' 'ABCDEFGHJKLMNPQRSTUVWXYZ' '23456789',
):
```
or
```python
cursor.execute("""
SELECT ...
""",
    [table name],
)
```

As far as I'm aware there is nothing else that could be marked with `fmt: off`, so after merging this PR I'm going to run `Black` on the entire Django code repository and add a single commit to the #15387.

### CODE REVIEW

**Comment by charettes** [comment]
Are we planning to add all of these pre-Black commits to `.git-blame-ignore-revs` as well?

**Comment by felixxm** [comment]
> Are we planning to add all of these pre-Black commits to `.git-blame-ignore-revs` as well?

IMO, that's the last one.

According to the DEP-8 only _"the whole-repo-refactor commit SHA should be added to the .gitblameskip or similar"_ (Black guarantees that it doesn't change the behavior of the code by checking that processing a file doesn't change its AST). Theoretically they can introduce a regression so I'm not sure if we should be doing this :thinking:

File: django/template/loader_tags.py [hunk 1]
```diff
@@ -315,13 +315,15 @@ def do_include(parser, token):
...
+            raise TemplateSyntaxError(
+                'The %r option was specified more than once.' % option
+            )
         if option == 'with':
             value = token_kwargs(remaining_bits, parser, support_legacy=False)
             if not value:
-                raise TemplateSyntaxError('"with" in %r tag needs at least '
-                                          'one keyword argument.' % bits[0])
+                raise TemplateSyntaxError(
+                    '"with" in %r tag needs at least one keyword argument.' % bits[0]
```
Comment (smithdc1) [inline django/template/loader_tags.py:325]: You may say this is out of scope. But thought it worth noting this is the first case I have spotted which has the string in single quotes and double quotes in the string. It seems elsewhere the string the user sees uses single quotes.

File: django/templatetags/tz.py [hunk 2]
```diff
@@ -209,6 +209,7 @@ def get_current_timezone_tag(parser, token):
     # token.split_contents() isn't useful here because this tag doesn't accept variable as arguments
     args = token.contents.split()
     if len(args) != 3 or args[1] != 'as':
-        raise TemplateSyntaxError("'get_current_timezone' requires "
-                                  "'as variable' (got %r)" % args)
+        raise TemplateSyntaxError(
+            "'get_current_timezone' requires 'as variable' (got %r)" % args
```
Comment (smithdc1) [inline django/templatetags/tz.py:213]: Like this...

File: django/templatetags/i18n.py [hunk 3]
```diff
@@ -477,13 +477,15 @@ def do_block_translate(parser, token):
...
+            raise TemplateSyntaxError(
+                'The %r option was specified more than once.' % option
+            )
         if option == 'with':
             value = token_kwargs(remaining_bits, parser, support_legacy=True)
             if not value:
-                raise TemplateSyntaxError('"with" in %r tag needs at least '
-                                          'one keyword argument.' % bits[0])
+                raise TemplateSyntaxError(
+                    '"with" in %r tag needs at least one keyword argument.' % bits[0]
```
Comment (smithdc1) [inline django/templatetags/i18n.py:487]: This one has double quotes for the user as well.

**Comment by charettes** [comment]
I think that Github still doesn't support `.git-blame-ignore-revs` but from the perspective of someone that uses `git-blame` locally and their web interface extensively when triaging issues or trying to differentiate behavior between Django versions this change will be highly disruptive so we should rely on this feature as much as possible to reduce the friction formatting the whole base will add.

Just my two cents, maybe others feel differently about this subject.

File: tests/migrations/test_writer.py [hunk 4]
```diff
@@ -86,8 +86,7 @@ def test_empty_signature(self):
         self.assertEqual(imports, {'import custom_migration_operations.operations'})
         self.assertEqual(
             buff,
-            'custom_migration_operations.operations.TestOperation(\n'
-            '),'
+            'custom_migration_operations.operations.TestOperation(\n),'
```
Comment (smithdc1) [inline tests/migrations/test_writer.py:89]:
```suggestion
            'custom_migration_operations.operations.TestOperation(\n),',
```

File: tests/postgres_tests/test_operations.py [hunk 5]
```diff
@@ -53,8 +53,7 @@ def test_add(self):
         operation = AddIndexConcurrently('Pony', index)
         self.assertEqual(
             operation.describe(),
-            'Concurrently create index pony_pink_idx on field(s) pink of '
-            'model Pony'
+            'Concurrently create index pony_pink_idx on field(s) pink of model Pony'
```
Comment (smithdc1) [inline tests/postgres_tests/test_operations.py:56]:
```suggestion
            'Concurrently create index pony_pink_idx on field(s) pink of model Pony',
```

File: tests/test_utils/tests.py [hunk 6]
```diff
@@ -928,9 +927,7 @@ def test_parsing_errors(self):
 
     def test_escaped_html_errors(self):
         msg = (
-            '<p>\n<foo>\n</p>'
-            ' != '
-            '<p>\n&lt;foo&gt;\n</p>\n'
+            '<p>\n<foo>\n</p> != <p>\n&lt;foo&gt;\n</p>\n'
         )
```
Comment (smithdc1) [inline tests/test_utils/tests.py:934]: Will this fit on a single line?

File: tests/annotations/tests.py [hunk 7]
```diff
@@ -987,8 +987,7 @@ def test_datetimes_alias(self):
 
     def test_aggregate_alias(self):
         msg = (
-            "Cannot aggregate over the 'other_age' alias. Use annotate() to "
-            "promote it."
+            "Cannot aggregate over the 'other_age' alias. Use annotate() to promote it."
         )
```
Comment (smithdc1) [inline tests/annotations/tests.py:992]: Does this fit into a single line?

**Review by smithdc1 — COMMENTED** [review]
Looks good, a few small notes. 

I've not run black on the entire code base, so I've not considered if there is anything "missing" from this.

File: tests/annotations/tests.py (hunk 7 above)
Comment (felixxm) [reply tests/annotations/tests.py:992]: nope

File: tests/test_utils/tests.py (hunk 6 above)
Comment (felixxm) [reply tests/test_utils/tests.py:934]: Yes, good catch :+1:

File: django/template/loader_tags.py (hunk 1 above)
Comment (felixxm) [reply django/template/loader_tags.py:325]: In these cases `black` will leave quotes untouched. I don't think it's worth changing.

**Comment by felixxm** [comment]
> I think that Github still doesn't support `.git-blame-ignore-revs` but from the perspective of someone that uses `git-blame` locally and their web interface extensively when triaging issues or trying to differentiate behavior between Django versions this change will be highly disruptive so we should rely on this feature as much as possible to reduce the friction formatting the whole base will add.
> 
> Just my two cents, maybe others feel differently about this subject.

Agreed, let's add all three to the `.git-blame-ignore-revs`.

**Review by carltongibson — APPROVED** [review]
These look good. Thanks @felixxm

**Comment by felixxm** [comment]
Thanks for checking :100:
